src/keyedit.cpp

```diff
@@ -434,35 +441,13 @@ void KeyMapForm::initialize(bool ingame)
 		}));
 		keyMapList->setGeometry(52, 10, KM_ENTRYW, 24 * KM_ENTRYH);
 
-		W_BUTINIT sButInit;
-
-		sButInit.formID		= KM_FORM;
-		sButInit.style		= WBUT_PLAIN | WBUT_TXTCENTRE;
-		sButInit.width		= KM_W;
-		sButInit.FontID		= font_regular;
-		sButInit.x		= 0;
-		sButInit.height		= 10;
-		sButInit.pDisplay	= displayTextOption;
-		sButInit.initPUserDataFunc = []() -> void * { return new DisplayTextOptionCache(); };
-		sButInit.onDelete = [](WIDGET *psWidget) {
-			assert(psWidget->pUserData != nullptr);
-			delete static_cast<DisplayTextOptionCache *>(psWidget->pUserData);
-			psWidget->pUserData = nullptr;
-		};
-
-		sButInit.id			= KM_RETURN;
-		sButInit.y			= KM_H - 32;
-		sButInit.pText		= _("Resume Game");
+		addButton(KM_GO_BACK, KM_H - 40, "Go Back");
```

Comment:
@past-due Will this affect translation if the `_()` is inside the function addButton?

Reply:
@KJeff01 Yes, I believe string extraction requires the actual string to be surrounded by `_()` (not just a variable containing the string). So the use of `_()` should be moved outside of `addButton`. (Good catch!)

Reply:
You are correct, but how it works in `multiint.cpp:1177`? There is a line like `_(limitIcons[i].desc)`

Reply:
> You are correct, but how it works in `multiint.cpp:1177`? There is a line like `_(limitIcons[i].desc)`

It would appear that the strings are added separately for extraction in the definition of `static const LimitIcon limitIcons[]`.
ex: `N_("Tanks disabled!!")`

Reply:
@Perl99 You'll want to do the same thing for the `addButton`s in `musicmanager.cpp` as well.